cli/src/CLIArgsParser.cpp

```diff
@@ -14,9 +14,7 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 ********************************************************************************/
-#include "OpenSpaceNetArgs.h"
-
-#include "OpenSpaceNet.h"
+#include "CLIArgsParser.h"
```

Comment:
The class/file name is against the naming conventions. Rename to `CliArgsParser.h`